h2hTester.js

```diff
@@ -1,27 +1,26 @@
-const fs = require('fs')
-const dotenv = require('dotenv')
 const Client = require('ssh2-sftp-client');
-
-// open sftpGo server
-
-dotenv.config()
+const fs = require('fs')
 
 const sftp = new Client();
 
-const passphrase = process.env.PASSPHRASE
-const homePath = process.env.HOME_PATH
-
-fileToGet = Math.floor(Math.random() * 2 + 1)
+const sftpTestConnection = (configObject) => {
+  let connection
+  sftp.connect(configObject)
```

Comment:
Nice start !!!

Let's try to do it using async await .. 

That way you'll see how you code is much simpler .. for example you will not need connection variable at all